dockers/docker-dhcp-server/cli/config/plugins/dhcp_server.py

```diff
@@ -1,13 +1,371 @@
 import click
 import utilities_common.cli as clicommon
 
+import ipaddress
+
 
 @click.group(cls=clicommon.AbbreviationGroup, name="dhcp_server")
 def dhcp_server():
     """config DHCP Server information"""
     pass
 
 
+@dhcp_server.group(cls=clicommon.AliasedGroup)
+def ipv4():
+    """Show ipv4 related dhcp_server info"""
+    pass
+
+
+@dhcp_server.group(cls=clicommon.AliasedGroup)
+def ipv6():
+    """Show ipv6 related dhcp_server info"""
+    pass
+
+
+def sanity_check_add(mode, lease_time, infer_gw_nm, gateway, netmask, dhcp_interface):
+    if mode != "PORT":
+        raise Exception("Only mode PORT is supported")
```

Comment:
May be use ctx.fail instead raising exception is better? https://github.com/sonic-net/sonic-buildimage/blob/7eb4872766df5d19b09bdc4089d77536c0b800d1/dockers/docker-dhcp-relay/cli/config/plugins/dhcp_relay.py#L19